doc/test-extensions.md

```diff
@@ -0,0 +1,157 @@
+# Testing Mayastor Extensions
+
+In order to test Mayastor, you'll need to be able to [**run Mayastor**][doc-run],
+follow that guide for persistent hugepages & kernel module setup.
+
+Or, for ad-hoc:
+
+- Ensure at least 512 2MB hugepages.
```

Comment:
we recommend minimum 1024 2MiB hugepages typically

Reply:
Existing docs mentions 512, so guess we should change it

Reply:
> Existing docs mentions 512, so guess we should change it

Here we say 1024: https://openebs.io/docs/4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-installation#preparing-the-cluster

Reply:
https://github.com/openebs/mayastor/blob/17488f4a7da3d44a2ea2b86ac0b2df7c57b02614/doc/test.md?plain=1#L8

Reply:
hmm I wonder if actually neither is sufficient, some tests use a lot of IO engines and run IO.
Perhaps we ought to run the tests with 1024 and check if it's enough.

Reply:
Alright, looks like for testing you actually need 3584 for the rust tests and 2560 for the bdd tests.